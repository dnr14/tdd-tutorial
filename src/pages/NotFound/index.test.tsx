import { screen, render } from '@testing-library/react';
import 'jest-styled-components';
import { NotFound } from '.';

describe('<NotFound>', () => {
  it('<NotFound> 컴포넌트 랜더링', () => {
    const { container } = render(<NotFound />);

    const text = screen.getByText('Not Found');
    expect(text).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
