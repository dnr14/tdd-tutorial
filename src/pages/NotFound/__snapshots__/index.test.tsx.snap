// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound> <NotFound> 컴포넌트 랜더링 1`] = `
.c0 {
  font-size: 20px;
}

<div>
  <div
    class="c0"
  >
    Not Found
  </div>
</div>
`;
